vendor/modules.txt

```diff
@@ -206,6 +206,7 @@ github.com/openshift/library-go/pkg/operator/resource/resourcehelper
 github.com/openshift/library-go/pkg/operator/resource/resourcemerge
 github.com/openshift/library-go/pkg/operator/resource/resourceread
 github.com/openshift/library-go/pkg/operator/resourcesynccontroller
+github.com/openshift/library-go/pkg/operator/staticresourcecontroller
```

Comment:
per discussion in office hours, @adambkaplan believes he can do some git refactoring to get this file's change minimally out of the main code change commit